components/benefit_sponsors/app/models/benefit_sponsors/benefit_sponsorships/benefit_sponsorship.rb

```diff
@@ -473,8 +545,8 @@ def benefit_applications_by(ids)
     # Open enrollment can be extended only for recent applications (i.e, upto 6 months old)
     # Benefit Application's with overlapping coverage can't be extended
     def oe_extendable_benefit_applications
-      benefit_applications.where(:"effective_period.min".gte => TimeKeeper.date_of_record.beginning_of_month).select do |benefit_application|
-        benefit_application.may_extend_open_enrollment? && !overlapping_coverage_exists?(benefit_application)
+      benefit_applications.select do |benefit_application|
+        benefit_application.start_on >= TimeKeeper.date_of_record.beginning_of_month && benefit_application.may_extend_open_enrollment? && !overlapping_coverage_exists?(benefit_application)
```

Comment:
benefit_application.start_on do we need convert to ```benefit_application.start_on.to_date```

Reply:
i don't think we need to do this since start_on is already a Date.